qlib/contrib/model/pytorch_tabnet.py

```diff
@@ -401,7 +397,7 @@ class FinetuneModel(nn.Module):
     """
 
     def __init__(self, input_dim, output_dim, trained_model):
-        super().__init__()
+        super(FinetuneModel, self).__init__()
```

Comment:
This is not necessary in Python3

Reply:
Should we follow PyTorch style for these PyTorch models? From FB's official implementation of different PyTorch models (https://github.com/pytorch/vision/blob/master/torchvision/models/googlenet.py#L73), they all use such style.

Reply:
Same as below.

Reply:
PyTorch wrote like this since it wants to be applicable for both Python 2.x and Python 3.x. [Link](https://stackoverflow.com/questions/61288224/why-not-super-init-model-self-in-pytorch). Since Python 2.x is no longer supported and Qlib is only available in Pyhon 3, this is actually useless.

Reply:
gotit~